keps/sig-storage/1979-object-storage-support/kep.yaml

```diff
@@ -13,12 +11,17 @@ reviewers:
   - "@alarge"
   - "@erinboyd"
   - "@guymguym"
-  - "@rsquared"
-  - "@krishchow"
+  - "@thockin"
 approvers:
   - "@saad-ali"
   - "@xing-yang"
+  - "@msau42"
+  - "@thockin"
 editor: TBD
 creation-date: 2019-11-25
-last-updated: 2020-09-10
-status: provisional
+last-updated: 2022-04-06
+status: alpha
+stage: alpha
+latest-milestone: v1.24
+milestone:
+  alpha: "v1.24"
```

Comment:
I'm not sure what the best way is to track out of tree features, but this should probably be 1.25 since we are almost at the end of the 1.24 cycle.

Reply:
I'll update it to v1.25